code/controllers/subsystem/mapping.dm

```diff
@@ -327,3 +330,12 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 
 		shelter_templates[S.shelter_id] = S
 		map_templates[S.shelter_id] = S
+
+/datum/controller/subsystem/mapping/proc/preloadRandomRoomTemplates()
+	for(var/item in subtypesof(/datum/map_template/random_room))
+		var/datum/map_template/random_room/room_type = item
+		if(!(initial(room_type.mappath)))
+			continue
```

Comment:
Shouldn't this be an error?

Reply:
Space and Lava ruins use the same check and both just "continue" if it fails.